internal/provider/resources/install/common.go

```diff
@@ -52,7 +59,33 @@ func (i *Install) reportConversionError(header string, subheader string, value a
 }
 
 func (i *Install) itemPath(id string) string {
-	return fmt.Sprintf("integrations/%s/config/%s/%s", i.Integration, i.Component, id)
+	return fmt.Sprintf("%s/%s/%s", i.itemBasePath(), i.Component, id)
+}
+
+func (i *Install) itemBasePath() string {
+	return fmt.Sprintf("integrations/%s/config", i.Integration)
+}
+
+// Places the item in the "stage" state in P0.
+// To use, the item's TFSDK model must be passed. For example:
+//
+//	var data ItemConfigurationModel
+//	var json ConfigurationApiResponseJson
+func (i *Install) Upsert(ctx context.Context, diags *diag.Diagnostics, plan *tfsdk.Plan, state *tfsdk.State, json any, model any) {
```

Comment:
```suggestion
func (i *Install) Stage(ctx context.Context, diags *diag.Diagnostics, plan *tfsdk.Plan, state *tfsdk.State, json any, model any) {
```
might be more descriptive